aten/src/ATen/core/opschema/layer_norm.cpp

```diff
@@ -1,4 +1,25 @@
 #include <ATen/core/opschema/layer_norm.h>
 #include <ATen/core/dispatch/OpSchemaRegistration.h>
 
-C10_DEFINE_OP_SCHEMA(c10::core::opschema::LayerNorm);
+namespace c10 {
+namespace core {
+namespace opschema {
+  // TODO Parse schema string instead of creating FunctionSchema manually
+  C10_DEFINE_OP_SCHEMA(LayerNorm, FunctionSchema(
+      "caffe2::layer_norm_dont_use_this_op_yet",
+      (std::vector<c10::Argument>{
```

Comment:
I'm surprised an initializer list isn't good enough here

Reply:
I'm not entirely sure why tbh, but it isn't according to the compiler.